### PR TITLE
i596 user-mngement-resource

### PR DESCRIPTION
**Pull request by petra-sargent**
…ment.

Signed-off-by: Petra Sargent <psargent@redhat.com>

### What does this PR do?

This PR contains the resource-managemnt.adoc file that was part of PR #508. The old PR #508 contains 3 files and will be closed. It is easier to have only one file per PR.

### What issues does this PR fix or reference?
#508

### CODE REVIEW

**Comment by petra-sargent** [comment]
@robin-owens @rkratky @preeticp Please review. Thanks!

File: src/main/pages/user-management/resource-management.adoc [hunk 1]
```diff
@@ -11,77 +11,82 @@ folder: user-management
 [id="overview"]
 == Overview
 
-Resource API is designed to help control resources that Che users consume. Che admins can set caps for free resources to each resource type, as well as for each particular account.
+The Resource API manages the resources that are utilized by Che users. The Che administrators set the limits on the amount of resources available for each user acount.
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: typo-account
Original content: Info about resource type not included. Suggest ...available for each resource type and each user...

File: src/main/pages/user-management/resource-management.adoc [hunk 2]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
 
-There are two kinds of accounts which may be used in Che:
+There are two kinds of accounts used in Che:
 
-* _personal_ - an account that belongs to a user. Only one user can be related to this type of account and consume resources provided to this account.
-* _organizational_ - account that belongs to an link:organizations.html[organization]. This type of account allows consuming related resources for each organization member. The organization can have sub-organizations and distribute resources between them.
+* _personal_ - This account belongs to a user. Only one user per account.
+* _organizational_ - This account belongs to an link:organizations.html[organization]. This type of account allows each member of the organization to use resources. Resources are distributed between organizations and sub-organizations.
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: ...to use its resources.
distributed between an organization and its sub-organizations.

File: src/main/pages/user-management/resource-management.adoc [hunk 3]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
 
-Resource API is designed to help control resources that Che users consume. Che admins can set caps for free resources to each resource type, as well as for each particular account.
+The Resource API manages the resources that are utilized by Che users. The Che administrators set the limits on the amount of resources available for each user acount.
 
-There are two kinds of accounts which may be used in Che:
+There are two kinds of accounts used in Che:
 
-* _personal_ - an account that belongs to a user. Only one user can be related to this type of account and consume resources provided to this account.
-* _organizational_ - account that belongs to an link:organizations.html[organization]. This type of account allows consuming related resources for each organization member. The organization can have sub-organizations and distribute resources between them.
+* _personal_ - This account belongs to a user. Only one user per account.
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: Only one user per account - missing verb / Only one user can utilize resources provided to the account.

File: src/main/pages/user-management/resource-management.adoc [hunk 4]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
+* _organizational_ - This account belongs to an link:organizations.html[organization]. This type of account allows each member of the organization to use resources. Resources are distributed between organizations and sub-organizations.
 
-Since workspaces are the critical for users in the development flow, resources are mostly related to workspaces and runtimes.
+The amount and usage refers to the amount and usage of resources for workspaces and runtimes.
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: Resource usage mostly refers to resources used by workspaces and runtimes in the development flow.

File: src/main/pages/user-management/resource-management.adoc [hunk 5]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
 
-Multi-user Che supports four types of resources:
+Multi-user Che supports the following types of resources:
 
-* *RAM* - defines the amount of RAM which can be used by running workspaces at the same time;
-* *Timeout* - defines the period of time that is used to control idling of user workspaces;
-* *Runtime* - defines the amount of workspaces which user can run at the same time;
-* *Workspace* - defines the amount of workspaces which user can have at the same time.
+* *RAM* -  Amount of RAM that workspaces use.
+* *Timeout* - Period of time that workspaces are idle.
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: As per the original, the period of time is used to control the idling...but I think here the meaning seems to have changed.

File: src/main/pages/user-management/resource-management.adoc [hunk 6]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
+* *Runtime* - Number of workspaces that users can run at the same time.
+* *Workspace* - Number of workspaces that users can have at the same time.
 
 [id="resource-api"]
 == Resource API
 
 *Total resources*
 
-`GET resource/${accountId}:` allows getting information on total resources list that is allowed to use by a specified account;
+`GET resource/${accountId}:` Gets the total resources allowed by an account;
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: ...the list of total resources an account can use.

File: src/main/pages/user-management/resource-management.adoc [hunk 7]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
 
 *Available resources*
 
-`GET resource/${accountId}/available:` allows getting information on available resources list that is not used, so if there are no used resources at the moment of the request, the result must equal total resources, otherwise, the result must contain deduction of used resources from total resources list;
+`GET resource/${accountId}/available:` Gets the resources that are available and not used by an account. If no resources are used, the available resources equal total resources.  If resources are used, the available resources equals total resources minus used resources.
 
 *Resource details*
 
-`GET resource/{accountId}/details:` allows getting detailed information about resource list that is provided to the specified account. The result of the request contains information about resource providers and the beginning/end of period of use.
+`GET resource/{accountId}/details:` Gets detailed information about the resources that are available for an account. The detailed information includes:  resource providers, resource start time, and resource end time.
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: drop 'that are'
resource-usage start time and resource-usage end time

File: src/main/pages/user-management/resource-management.adoc [hunk 8]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
-== Resource distribution
+[id="distributing-resources"]
+== Distributing resources
 
-There are three ways to distribute resources to account:
+The following are ways to distribute resources to an account:
 
-* Che admin specifies default free resources limit for account by configuration;
-* Che admin overrides default free resources limit for account by resource-free API.
+* Che administrator specifies default free resources limit for account by configuration.
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc:56]: an account using the configuration. (Does configuration refer to the system configuration here or an API as in the next point? Would be good to mention ...the system configuration/the configuration API. )

File: src/main/pages/user-management/resource-management.adoc [hunk 9]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
-[id="configuration"]
-== Configuration
+* Che administrator overrides the default free resources limit for account by resource-free API.
 
-Che admin can limit how workspaces are created and the resources that these workspaces consume. Detailed information about each property can be found in https://github.com/eclipse/che/blob/master/dockerfiles/init/manifests/che.env#L538[che.env] file.
+[id="configuring-workspaces-and-resources"]
+== Configuring workspaces and resources
 
-See: link:docker-config.html[Docker] and link:openshift-config.html[OpenShift] configuration docs.
+The Che admininstrator can limit how workspaces are created and the resources that these workspaces consume. Detailed information about each property can be found in the https://github.com/eclipse/che/blob/master/dockerfiles/init/manifests/che.env#L538[che.env] file.
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: `che.env` use backticks

File: src/main/pages/user-management/resource-management.adoc [hunk 10]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
 |Property name |Default Value |Unit |Description
-|`CHE_LIMITS_USER_WORKSPACES_COUNT` |-1 |item |count of workspaces Che user is allowed to create
-|`CHE_LIMITS_USER_WORKSPACES_RUN_COUNT` |-1 |item |count of simultaneously running workspaces for Che user
-|`CHE_LIMITS_USER_WORKSPACES_RAM` |-1 |memory |amount of RAM all user’s workspaces can simultaneously consume
-|`CHE_LIMITS_ORGANIZATION_WORKSPACES_COUNT` |-1 |item |count of workspaces Organization members are allowed to create
-|`CHE_LIMITS_ORGANIZATION_WORKSPACES_RUN_COUNT` |-1 |item |count of simultaneously running workspaces by Organization members
-|`CHE_LIMITS_ORGANIZATION_WORKSPACES_RAM` |-1 |memory |amount of RAM all organizations’s workspaces can simultaneously consume
-|`CHE_LIMITS_WORKSPACE_IDLE_TIMEOUT` |-1 |minutes |timeout to idle inactive workspaces
-|`CHE_LIMITS_WORKSPACE_ENV_RAM` |16gb |memory |maximum amount of RAM that workspace environment can use at one
+|`CHE_LIMITS_USER_WORKSPACES_COUNT` |-1 |item |maximum number of workspaces that the Che user creates
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: a Che user can create

File: src/main/pages/user-management/resource-management.adoc [hunk 11]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
+|`CHE_LIMITS_USER_WORKSPACES_RUN_COUNT` |-1 |item |maximum number of simultaneously running workspaces for Che user
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: a Che user

File: src/main/pages/user-management/resource-management.adoc [hunk 12]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
+|`CHE_LIMITS_USER_WORKSPACES_RAM` |-1 |memory |maximum amount of RAM that workspaces use
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc:72]: can simultaneously use

File: src/main/pages/user-management/resource-management.adoc [hunk 13]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
+|`CHE_LIMITS_ORGANIZATION_WORKSPACES_COUNT` |-1 |item |maximum number of workspaces that members of an organization create
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: can create

File: src/main/pages/user-management/resource-management.adoc [hunk 14]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
+|`CHE_LIMITS_ORGANIZATION_WORKSPACES_RUN_COUNT` |-1 |item |maximum number of simultaneously running workspaces by members of an organization
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: maximum number of workspaces that members of an organization can simultaneously run

File: src/main/pages/user-management/resource-management.adoc [hunk 15]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
+|`CHE_LIMITS_ORGANIZATION_WORKSPACES_RAM` |-1 |memory |maximum amount of RAM that workspaces can use at the same time
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: workspaces from all organizations
This is important to distinguish from, workspaces belonging to a user/organization/all organizations.

File: src/main/pages/user-management/resource-management.adoc [hunk 16]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
+|`CHE_LIMITS_WORKSPACE_IDLE_TIMEOUT` |-1 |minutes |maxium number of minutes that workspaces are inactive
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: How about ...workspaces can stay inactive before they are idled?

File: src/main/pages/user-management/resource-management.adoc [hunk 17]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
+|`CHE_LIMITS_WORKSPACE_ENV_RAM` |16gb |memory |maximum amount of RAM that workspace environment use
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: can use simultaneously

File: src/main/pages/user-management/resource-management.adoc [hunk 18]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
 |===
 
 [id="unit-formats"]
-== Unit formats:
+== Unit formats
+
+The unit has the following formats:
+
+* `-1`: Unlimited value. Any operation, aggregation, and deduction of resources will return `-1`.
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: An unlimited value.

File: src/main/pages/user-management/resource-management.adoc [hunk 19]
```diff
@@ -11,77 +11,82 @@ folder: user-management
...
 
-The value `-1` mean that it is unlimited and any operation aggregation and deduction of resources will return `-1`.
+* `memory`:  A plain or fixed-point integer measured in bytes.
 
-* `memory` - measured in bytes. This value can be defined as a plain integer or as a fixed-point integer using one of following suffixes:
+ Memory uses one of the following suffixes:
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: Add '+' in the line above the table to align it under the above bullet point.

File: src/main/pages/user-management/resource-management.adoc [hunk 20]
```diff
@@ -93,43 +98,43 @@ The value `-1` mean that it is unlimited and any operation aggregation and deduc
 |`p` / `pb` / `pib` |peta bytes `1p` = `1024t`
 |===
 
-* `item` - the integer describing the number of objects;
-* `minutes` - the time frame, which is specified in integer value of minutes.
+* `item` - Integer describing the number of objects.
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: An integer
Same for the next point

File: src/main/pages/user-management/resource-management.adoc [hunk 21]
```diff
@@ -93,43 +98,43 @@ The value `-1` mean that it is unlimited and any operation aggregation and deduc
...
-`GET resource/free:` allows getting information on free resources list that is provided to this account;
+`GET resource/free:` Gets the resources that are available.
 
-`GET resource/free/{accountId}:` allows getting information on free resources list that is provided to specified account;
+`GET resource/free/{accountId}:` Gets the resources that are available for this account.
 
 *Set Free Resources*
 
-`POST resource/free:` allows setting resource caps for the specified user/organization account. This limits will override Сhe configuration and be used in all further operations with resources;
+`POST resource/free:` Sets the maximum amount of resources available for the user account. This number overrides the Сhe configuration. It will be used in all further operations with resources.
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: user or organization account

File: src/main/pages/user-management/resource-management.adoc [hunk 22]
```diff
@@ -93,43 +98,43 @@ The value `-1` mean that it is unlimited and any operation aggregation and deduc
...
 
 *Distributed Organization Resources*
 
-`GET organization/resource/{organizationId}:` allows getting information on total resources list that is provided to sub-organization by its parent organization;
+`GET organization/resource/{organizationId}:` Gets the resources that the parent organization provides to the sub-organization.
 
 *Sub-Organization Resources Cap*
 
-`GET organization/resource/{suborganizationId}/cap:` allows getting information on resources caps that are set for a sub-organization; By default, sub-organization is able to use all parent organization resources;
+`GET organization/resource/{suborganizationId}/cap:` Gets the maximum amount of resources that are available for a sub-organization; By default, sub-organizations use all the resources of the parent organization.
```
Comment (Preeticp) [inline src/main/pages/user-management/resource-management.adoc]: ...can use all the resources...

**Review by rkratky — APPROVED** [review]

**Pull request merged**